src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -224,6 +224,12 @@ class GoogleDataprocDAO(protected val leoServiceAccountEmail: WorkbenchEmail,
       Map.empty
 
     new SoftwareConfig().setProperties((authProps ++ workerProps).asJava)
+
+      // This gives us Spark 2.0.2. See:
+      //   https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions
+      // Dataproc supports Spark 2.2.0, but there are no pre-packaged Hail distributions past 2.1.0. See:
+      //   https://hail.is/docs/stable/getting_started.html
+      .setImageVersion("1.1")
```

Comment:
We could use Spark 2.2.0, but then we'd need to build Hail in the Dockerfile instead of using a pre-built version.